packages/webcert/src/store/patient/patientMiddleware.ts

```diff
@@ -56,15 +51,14 @@ const handleGetPatientError: Middleware<Dispatch> = ({ dispatch, getState }: Mid
   dispatch(throwError(error))
 }
 
-const handleChangePatient: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (action: AnyAction): void => {
-  dispatch(clearPatient())
-  action.payload.push('/create')
+const handleClearPatient: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (action: AnyAction): void => {
```

Comment:
This can be removed and the clearPatient() in the reducer can be used instead

Reply:
Done.